files/image_config/rsyslog/rsyslog.conf.j2

```diff
@@ -122,6 +123,11 @@ template(name="WelfRemoteFormat" type="string" string="%TIMESTAMP% id=firewall t
 action(type="omfwd" Target="{{ server }}" Port="{{ port }}" Protocol="{{ proto }}" Template="{{ template }}"{{ options }})
 {% endfor %}
 
+# update rsyslog counter
```

Comment:
For template purposes, I'd recommend moving the comment to the inside of the if-block, so that it's not rendered if the syslog counter feature is not enabled at all.